src/fiat/bank-authorities.md

```diff
@@ -83,47 +98,166 @@ curl -X POST "https://service.centrapay.com/api/bank-authorities" \
+| 403    | {% break _ BANK_AUTHORITY_LIMIT_EXCEEDED %}              | The account already has the max amount of bank authorities.                                                |
 | 403    | {% break _ BANK_AUTHORITIES_FOR_BANK_ACCOUNT_EXCEEDED %} | There are already two bank authorities for the provided bank account number, which is the maximum allowed. |
 
-## Get information about a bank authority
+<span id="direct-debit-authority"></span>
+## Adding a direct debit authority to a bank account **EXPERIMENTAL**
 
-{% endpoint GET https://service.centrapay.com/api/bank-authorities/${id} %}
+{% endpoint POST https://service.centrapay.com/bank-accounts/${bankAccountId}/direct-debit-authorities %}
+
+```sh
+curl -X PUT https://service.centrapay.com/api/bank-accounts/WRhAxxWpTKb5U7pXyxQjjY/direct-debit-authorities \
+  -H "x-api-key: 1234" \
+  -H "content-type: application/json" \
+  -d '{
+    "authorized": true,
+    "phoneNumber": "+64212345678",
+    "fullName": "John Doe",
+    "emailAddress": "john@doe.org"
+  }'
+```
+
+**Example response payload**
+
+```json
+{
+  "id": "WRhAxxWpTKb5U7pXyxQjjY",
+  "accountId": "Jaim1Cu1Q55uooxSens6yk",
+  "bankAccountNumber": "12-1234-1234567-123",
+  "bankAccountName": "John Doe",
+  "directDebitAuthorized": true,
+  "status": "created",
+  "verified": false,
+  "createdAt": "2020-06-12T01:17:46.499Z",
+  "createdBy": "crn:WIj211vFs9cNACwBb04vQw:api-key:MyApiKey",
+  "modifiedAt": "2020-06-12T01:17:46.499Z",
+  "modifiedBy": "crn:WIj211vFs9cNACwBb04vQw:api-key:MyApiKey",
+}
+```
+
+**Required Fields**
+
+Note, fields which have a star (✩) are required for Top Up. All fields below when specified are
+required together.
+
+|    Field     |  Type   |                    Description                                                                               |
+| :----------- | :------ | :----------------------------------------------------------------------------------------------------------- |
+| authorized   | Boolean | ✩ The user accepts our [Direct Debit terms][dd-terms]{:.external} and has authority to operate this account. |
+| phoneNumber  | String  | ✩ The user's phone number.                                                                                   |
+| fullName     | String  | ✩ The first and last name of the user.                                                                       |
+| emailAddress | String  | ✩ The user's email address.                                                                                  |
+
+**Error Responses**
+
+| Status |                           Code         |                          Description                               |
+| :----- | :------------------------------------- | :----------------------------------------------------------------- |
+| 403    | {% break _ BANK_AUTHORITY_FINALIZED %} | This bank authority cannot be changed as all fields have been set. |
```

Comment:
Can we change the error code to something like `BANK_AUTHORITY_ALREADY_DIRECT_DEBIT_AUTHORIZED`?

Reply:
Updated this :)

Reply:
@heymatthew we could shorten it to DIRECT_DEBIT_ALREADY_AUTHORIZED

Reply:
Done.